oauth.rst

```diff
@@ -1,131 +1,129 @@
-Partner Integrations: Public Documentation
-=========================================
+DOCK OAuth Integration Guide
+============================
 
-The OAuth 2.0 spec defines four roles:
+Initial Setup
+-------------
 
-- Resource owner
-- Resource server
-- Authorization server
-- Client
+Any new Partner application needs to be first set up with DOCK manually before any authentication or data exchange services can be used. Following are the one time set up steps for partners:
 
-The following document will try to explain how Dock approaches its OAuth2 implementation.
+- Provide basic info like your application's name, logo and description.
+- Provide these URLs: homepage, privacy policy and at least one callback URI. 
+The above information, except for the callback URIs, will be shown to the User at the Authorization Page. 
 
-The User
---------
-The OAuth 2.0 spec refers to the user as the "resource owner". The resource owner is the person who is giving access to some portion of their account. In our case this is a user in the Dock App. The resources in our case is data: personal information, contacts, work experience, etc. Any system that wants to act on behalf of the user must first get permission from them.
+- As a result of registering, you will be given a ``client_id`` and a ``client_secret`` that you will need to use to authenticate your requests to our API.  Please be aware that we do not store your ``client_secret``. You need to save it somewhere safe as soon as you get it since we cannot retrieve it back for you if you lose it. In that event a new set of client id/secret will have to be created.
```

Comment:
I'd suggest to remove the very last sentence here since we don't really need to create a new id, just a new secret.

Reply:
changing sentence to just mention about new secret